tools/manifest.json

```diff
@@ -57,5 +59,19 @@
 	},
 	"Clients": {
 		"owner": [9,9,9]
+	},
+	"Secrets": {
+		"web_cert_shared": {
+			"Shared": true,
+			"Size": 256,
+			"Type": "cert-ecdsa",
+			"Cert": {
+				"Subject": {
+					"CommonName": "localhost",
+					"DNSNames": "localhost"
```

Comment:
Is this working or does it have to be `["localhost"]`?

Reply:
worked for me, but let le me retest wit ["localhost"], which is probably the correct way of doing it.